docs/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 gooberjs
```

Comment:
Really small nit: I think the project name should be in here without the `js` part. What do you think?

Reply:
Good catch! Just removed the `js` part.